### PR TITLE
Avoid resolving Bogus names in RecognizePlaces, but do so in ResolveD…

### PR DESCRIPTION
**Pull request by Spaceenter**
…escription.

### CODE REVIEW

File: proto/recon.proto [hunk 1]
```diff
@@ -126,7 +126,12 @@ message BulkFindEntitiesResponse {
   repeated Entity entities = 1;
 }
 
-message RecognizePlacesRequest { repeated string queries = 1; }
+message RecognizePlacesRequest {
+  repeated string queries = 1;
+  // When set, bogus name should be resolved.
+  // This needs to be set for ResoveDescription, but not set for RecognizePlaces for sentences.
+  bool resolve_bogus_name = 2;
```
Comment (pradh) [inline proto/recon.proto:133]: nit: maybe call it `require_context_for_common_names`?  we internally call it bogus name.

File: internal/store/files/recog_place_store.go [hunk 2]
```diff
@@ -205,3 +216,23 @@ func loadAbbreviatedNames() (map[string][]string, error) {
...
+	}
+
+	res := map[string]struct{}{}
+
+	for _, record := range records {
+		if len(record) != 1 {
+			return nil, status.Errorf(codes.FailedPrecondition,
+				"Wrong BogusPlaceNames CSV record: %v", record)
+		}
+		res[strings.ToLower(record[0])] = struct{}{}
```
Comment (pradh) [inline internal/store/files/recog_place_store.go:234]: nit: Trim extra spaces?

**Review by pradh — APPROVED** [review]
Thanks!

File: internal/store/files/recog_place_store.go (hunk 2 above)
Comment (Spaceenter) [reply internal/store/files/recog_place_store.go:234]: done

File: proto/recon.proto (hunk 1 above)
Comment (Spaceenter) [reply proto/recon.proto:133]: Per offline discussion - making this an internal flag.

**Pull request merged**